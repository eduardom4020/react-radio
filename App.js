import React from 'react';
import { RadiosList } from '~/src/modules';
import { Background } from '~/src/components';
import { DARK_GREY } from '~/src/constants/colors';

const App = () => {
    return (
        <Background color={DARK_GREY} >
            <RadiosList />
        </Background>
    );
};

export default App;