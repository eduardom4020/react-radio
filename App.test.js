import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '~/App';

describe('App (root)', () => {
    test('Render?', () => {
        const component = shallow(
            <App />
        );

        expect( toJson(
            component, {
                noKey: false,
                mode: 'deep'
            }
        ) ).toMatchSnapshot();
    });
});