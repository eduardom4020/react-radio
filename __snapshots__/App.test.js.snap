// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App (root) Render? 1`] = `
<Styled(View)
  color="#2e2e3a"
>
  <RadioListViewDefault />
</Styled(View)>
`;
